doc/whats-new.rst

```diff
@@ -119,9 +118,22 @@ New Features
 - Raise more informative error when decoding time variables with invalid reference dates.
   (:issue:`5199`, :pull:`5288`). By `Giacomo Caria <https://github.com/gcaria>`_.
 
+Breaking changes
```

Comment:
this is in the wrong place! cc @TomNicholas

Reply:
Oops - should now be fixed in `main`